ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -81,19 +85,33 @@ usage() {
 defaultVars() {
 	case "$ARCHITECTURE" in
 		"s390x" | "S390X" | "S390x" )
-			echo "s390x selected"; ARCHITECTURE=S390X;;
+			ARCHITECTURE=S390X;;
 		"aarch64" | "arm64" | "ARM64" )
-			echo "aarch64 selected"; ARCHITECTURE=AARCH64;;
+			ARCHITECTURE=AARCH64;;
 		"ppc64le" | "ppc64" | "PPC64LE" | "PPC64" )
-			echo "ppc64le selected"; ARCHITECTURE=PPC64LE;;
+			ARCHITECTURE=PPC64LE;;
 		"arm32" | "ARM32" | "armv7l" | "ARMV7L")
-			echo "arm32 selected"; ARCHITECTURE=ARM32;;
+			ARCHITECTURE=ARM32;;
 		"RISC-V" | "riscv" | "risc-v" | "RISCV" )
-			echo "riscv selected"; ARCHITECTURE=RISCV;;
+			ARCHITECTURE=RISCV;;
 		"" )
 			echo "Please input an architecture to test"; exit 1;;
 		*) echo "Please select a valid architecture"; showArchList; exit 1;;
 	esac
+
+	case "$OS" in
+		"debian8" | "Debian8" | "deb8" )
+			echo "DEBIAN8 selected for $ARCHITECTURE"; OS=DEBIAN8;;
```

Comment:
```suggestion
			echo "Debian8 selected for \$ARCHITECTURE"; OS=DEBIAN8;;
```

Reply:
Ive kept the os in caps to keep it consistent with the how the architecture is displayed

Reply:
Ah, right! Okay, yep that makes more sense